/* Copyright 2013 MaidSafe.net limited

This MaidSafe Software is licensed under the MaidSafe.net Commercial License, version 1.0 or later,
and The General Public License (GPL), version 3. By contributing code to this project You agree to
the terms laid out in the MaidSafe Contributor Agreement, version 1.0, found in the root directory
of this project at LICENSE, COPYING and CONTRIBUTOR respectively and also available at:

http://www.novinet.com/license

Unless required by applicable law or agreed to in writing, software distributed under the License is
distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
implied. See the License for the specific language governing permissions and limitations under the
License.
*/

package maidsafe.lifestuff;

message PassportData {
  required bytes serialised_keyring = 1;
}

message UserData {
  required bytes unique_user_id = 1;
  required bytes root_parent_id = 2;
  required bytes vault_path = 3;
  required int64 max_space = 4;
  required int64 used_space = 5;
}

message DataAtlas {
  optional UserData user_data = 1;
  required PassportData passport_data = 2;
  required bytes timestamp = 3;
}
